agent/bootstrap_agent.go

```diff
@@ -32,6 +32,10 @@ func (a *bootstrapAgent) Run() {
 	a.Process()
 }
 
+var startBootstrap = func(a *bootstrapAgent, pc pfmodel.Container) {
+	go a.bootstrapContainer(pc)
```

Comment:
this is go routine so that the bootstrap can be done in parallel, right?

Reply:
should we limit the parallelization by the way?

Reply:
> this is go routine so that the bootstrap can be done in parallel, right?
yes

> should we limit the parallelization by the way?
yes we can setup the limit. will create an env variable for this

Reply:
Updated Mas.